datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -49,8 +49,10 @@ def main(
 
     logging.info("Creating 'files' folder")
     pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
-
-    if "individuals" not in pipeline_name:
+    if (
+        "individuals" not in pipeline_name
+        and "other_committee_tx_2020" not in pipeline_name
+    ):
         logging.info(f"Downloading file from {source_url}...")
```

Comment:
Move logging to respective function definition.

Reply:
done